js/index.js

```diff
@@ -1 +1,42 @@
 // Your code goes here
+
+
+// mouseover
+//  keydown
+//  wheel
+//  drag / drop
+//  load
+//  focus
+//  resize
+//  scroll
+//  select
+//  dblclick
+
+const signBtn = document.querySelectorAll(".btn");
+
+// signBtn.forEach(button => {
+//     button.addEventListener ("click", () => {
+//         event.target.style.backgroundColor = 'blue';
+//     }
+// })
+
+
+const goCrazy = document.querySelectorAll(".container home .content-section .text-content p");
```

Comment:
```
Uncaught TypeError: Cannot read property 'addEventListener' of undefined
```

I think it's because of `container home` being called. Just choose between one or the other. Also, you need a `.` on `home`